apps/render_cf.py

```diff
@@ -158,6 +165,47 @@ def render_templates(job_types, security_environment, api_name):
         template_file.with_suffix('').write_text(output)
 
 
+def get_compute_environments(job_types: dict, compute_env_file: Optional[Path]) -> list[dict]:
+    compute_envs = []
+    compute_env_names = set()
```

Comment:
How do you feel about this as a way to ensure that an error is raised if someone tries to define a `Default` compute environment?
```suggestion
    compute_env_names = {'Default'}
```

Reply:
Meh, I think that's a bit confusing since we later construct a `compute_env_names` variable which doesn't include `'Default'`. Maybe let's add an explicit check for this below.